### DISCUSSION TITLE
PlanetScale alternative

### DISCUSSION THREAD

**Discussion by janklarer** [mitchmac/ServerlessWP#66] [General]
Is there any other provider instead of planetscale? because there isnt a free plan available anymore on planet scale;/

**Comment by awan-gh** [comment]
i've been using [aiven.io](https://aiven.io/mysql) for the longest time :)

**Reply by MatteoGeusa** [reply]
i cant connect aiven to vercel, i have this error
![image](https://github.com/user-attachments/assets/71b857a8-d72f-4f9e-b69f-c4acaf0ff7ed)
